### ISSUE TITLE
Version 3.2 issues

### ISSUE THREAD

**Issue by gabriel-tandil** [anselm94/Torchie-Android#26]
Hello! Nice app.
Since the last update from f-droid (3.2) the torch goes off by itself but the waiting time is disabled and sometimes does not turn on when pressing the buttons.
The previous version worked perfect
Android 4.4

**Comment by gabriel-tandil** [comment]
one more tip in case it helps...
The moment when the torch goes out on its own coincides with when the screen is turned off due to inactivity.
After this the volume buttons to turn it on no longer respond.

I repeat that this did not happen with the previous version.

**Comment by anselm94** [comment]
@gabriel-tandil Just created a release 3.2.1 and should be available in F-droid by tomorrow. I don't have a Android 4.4 device to verify. Please let me know, if it is reproducible in 3.2.1.

**Comment by gabriel-tandil** [comment]
Hello!
Torchie has just been updated from FDroid.
The problem that it turned off and then couldn't turn back on is fixed <3

The only thing that I feel different from the previous functional version is that I can't turn off the flashlight with the screen off (before I could). But although this was comfortable it's not really a serious problem. Even with that previous version on other phones that couldn't be done. 

Thank you, greetings

Gabriel

**Comment by anselm94** [comment]
@gabriel-tandil Glad to hear it works! Does this work everytime except while in screen off in Android 4.4?

**Comment by gabriel-tandil** [comment]
Yes. Everything works properly now except with the display turned off. 
Previously, even with the "screen off" option disabled, the flashlight could be turned off with the screen off.

Greetings and congratulations again for the app.

Gabriel
